doc/lib.rst

```diff
@@ -595,9 +573,21 @@ Network Programming and Internet Protocols
   Wrapper for OpenSSL.
 
 
-
 Unstable
 ========
```

Comment:
not a fan of this category, IMO it's better to move those to a topic section (TBD) and indicate in the module description that the module is unstable/experimental